[2023-10-16T17:31:28.344+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup.taskA scheduled__2023-09-30T12:00:00+00:00 [queued]>
[2023-10-16T17:31:28.346+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup.taskA scheduled__2023-09-30T12:00:00+00:00 [queued]>
[2023-10-16T17:31:28.346+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-16T17:31:28.349+0100] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): taskA> on 2023-09-30 12:00:00+00:00
[2023-10-16T17:31:28.351+0100] {standard_task_runner.py:57} INFO - Started process 18316 to run task
[2023-10-16T17:31:28.354+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup', 'taskA', 'scheduled__2023-09-30T12:00:00+00:00', '--job-id', '164', '--raw', '--subdir', 'DAGS_FOLDER/catchup.py', '--cfg-path', '/var/folders/_3/bj8r7kz51nz4g7hl_dpr6jdw0000gn/T/tmpx4q2db5f']
[2023-10-16T17:31:28.355+0100] {standard_task_runner.py:85} INFO - Job 164: Subtask taskA
[2023-10-16T17:31:28.368+0100] {task_command.py:416} INFO - Running <TaskInstance: cron_catchup.taskA scheduled__2023-09-30T12:00:00+00:00 [running]> on host 192.168.1.112
[2023-10-16T17:31:28.387+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='workflow' AIRFLOW_CTX_DAG_ID='cron_catchup' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2023-09-30T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-09-30T12:00:00+00:00'
[2023-10-16T17:31:28.387+0100] {subprocess.py:63} INFO - Tmp dir root location: /var/folders/_3/bj8r7kz51nz4g7hl_dpr6jdw0000gn/T
[2023-10-16T17:31:28.388+0100] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "task A executed"']
[2023-10-16T17:31:28.391+0100] {subprocess.py:86} INFO - Output:
[2023-10-16T17:31:28.392+0100] {subprocess.py:93} INFO - task A executed
[2023-10-16T17:31:28.392+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2023-10-16T17:31:28.398+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=cron_catchup, task_id=taskA, execution_date=20230930T120000, start_date=20231016T163128, end_date=20231016T163128
[2023-10-16T17:31:28.415+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-16T17:31:28.424+0100] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
